src/amo/pages/SearchTools/index.js

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
+import NotFoundPage from 'amo/pages/ErrorPages/NotFoundPage';
```

Comment:
We try to keep these in alphabetical order, so this should be moved to the line above `import { sendServerRedirect } from 'amo/reducers/redirectTo';`

Reply:
Sure, I'll be making an attempt at a test soon.